src/gtfs/write.rs

```diff
@@ -73,11 +73,11 @@ fn get_first_comment_name<T: objects::CommentLinks>(
     obj: &T,
     comments: &CollectionWithId<objects::Comment>,
 ) -> Option<String> {
-    comments
-        .iter_from(obj.comment_links())
-        .map(|c| &c.name)
+    obj.comment_links()
+        .iter()
+        .filter_map(|comment_id| comments.get(comment_id))
+        .map(|cmt| cmt.name.clone())
```

Comment:
It looks like cloning here will clone every name, while previously only the selected one was cloned.